### ISSUE TITLE
PS5 mask type problem

### ISSUE THREAD

**Issue by ThisRock84** [LightningMods/Itemzflow#67]
Hello,
After trying the new etaHEN 1.8, I noticed that some patches work, and some have this issue: https://imgur.com/a/ihIB8X2

It might be due to illusion0001's newer patch format that uses Type="mask" instead of Type="bytes", but I'm not certain.

All patch files written as Mask are also problematic for me. I changed the mask text to AppVer=‘00.34’ for example, but even so, the patch still writes an error in the notify of eboot waiting for eboot and does not work.

**Comment by LightningMods** [comment]
Hello, yes currently the mask type is not supported however @illusion0001 said he will add support so I'll keep this issue open until it's added

**Comment by ThisRock84** [comment]
Thank you for your prompt response and for keeping the issue open. I appreciate the information and look forward to the support for the "mask" type being added in the future. Your efforts in improving etaHEN are greatly valued by the community!

**Comment by jackfavvv** [comment]
> Hello,
> 
> While trying out the new etaHEN 1.8, I noticed that some patches work, while others show this issue: https://imgur.com/a/ihIB8X2
> 
> This might be due to illusion0001's new patch format, which uses Type="mask" instead of Type="bytes", but I'm not entirely sure.

I Just noticed, but did you literally just copy & pasted that from a question that I asked awhile back? 0_o
That's definitely my image from FATAL FRAME: Mask of the Lunar Eclipse.

**Comment by ThisRock84** [comment]
> > Hello,
> > While trying out the new etaHEN 1.8, I noticed that some patches work, while others show this issue: https://imgur.com/a/ihIB8X2
> > This might be due to illusion0001's new patch format, which uses Type="mask" instead of Type="bytes", but I'm not entirely sure.
> 
> I Just noticed, but did you literally just copy & pasted that from a question that I asked awhile back? 0_o That's definitely my image from FATAL FRAME: Mask of the Lunar Eclipse.

Absolutely! :) I combined the question you previously asked on Reddit with the work I've been doing, to create a single, comprehensive post for easier reference. The first image is part of your original post, while the others are from my own testing. Thank you for your keen observation, I appreciate the attention to detail!

**Comment by f0li083** [comment]
> In the meantime, I've found a way to convert Type="mask" patches.

Hi Jack, can you provide some insight on patch conversion please or point me in the right direction so I can get started? Thanks.

**Comment by jackfavvv** [comment]
> > In the meantime, I've found a way to convert Type="mask" patches.
> 
> Hi Jack, can you provide some insight on patch conversion please or point me in the right direction so I can get started? Thanks.

Hello f0li083,
That might be a bit difficult since this isn't like an auto backport tool where you can just press a button, and the whole process is done automatically. You need to manually edit hex values from the eboot.bin, and you need to work out the patterns for the byte values yourself. 

Unfortunately, I don't think I can point you to other sources since I might be the only one doing this. Wait, there is another user, edmaren. Try messaging him.
